subiquity/common/types.py

```diff
@@ -754,12 +754,25 @@ class ADConnectionInfo:
     password: str = attr.ib(repr=False, default="")
 
 
-class ADValidationResult(enum.Enum):
+class AdAdminNameValidation(enum.Enum):
```

Comment:
Usually, we try to give sensible values for the enums fields that are serialized / deserialized and transit over HTTP.
Right now, the different values will be represented as integers, so this isn't too user friendly.